conformance.adoc

```diff
@@ -632,4 +632,101 @@ in the file.
+  and each **`interpolation_zone_dimension`** token specifies a unique
+  interpolation zone dimension. The tie point interpolation dimensions
+  and interpolation zone dimensions must not be dimensions of the
+  parent data variable.
+  
+* When attached to a data variable, the type of the
+  **`tie_point_indices`** attribute is a string whose value is a list of
+  blank separated word pairs of the following form:
+  **`interpolation_dimension: tie_point_index_variable`**. The
+  **`interpolation_dimension`** tokens specify the same interpolation
+  dimensions as the **`tie_point_dimensions`**, and each
+  **`tie_point_index_variable`** token specifies a tie point index
+  variable that must exist in the file.
+
+* A tie point index variable must be a one-dimensional variable with
+  an integer data type.
+
+* The dimension of a tie point index variable must be a tie point
+  interpolation dimension identified by the **`tie_point_dimensions`**
+  attribute.
+
+* The values of a tie point index variable must be non-negative
+  integers. The first value must be zero, and each subsequent value
+  must be greater than or equal to the previous value. If a value
+  differs by zero or one from its previous value, then it must differ
+  by two or more from its subsequent value.
+
+* When attached to a data variable, the type of the **`tie_points`**
+  attribute is a string whose value is a list of blank separated word
+  groups of the following form, in which brackets indicate optional
+  text: **`tie_point_variable: [tie_point_variable: ...]
+  interpolation_variable`**. Each **`tie_point_variable`** token
+  specifies a tie point variable that must exist in the file, and each
+  **`interpolation_variable`** token specifies an interpolation
+  variable that must exist in the file.
+
+* The tie point variables associated with each
+  **`interpolation_variable`** token must all span the same
+  dimensions, which comprise a subset of zero or more dimensions of
+  the parent data variable with the addition of at least one tie point
+  interpolation dimension identified by the **`tie_point_dimensions`**
+  attribute. A tie point variable must not span both a tie point
+  interpolation dimension and its corresponding interpolation
+  dimension, as defined by the **`tie_point_dimensions`** mapping.
+
+* An interpolation variable must have one of the string-valued
+  attributes **`interpolation_name`** or
+  **`interpolation_description`**, but not both. The legal values for
+  the **`interpolation_name`** attribute are contained in Appendix J.
+
+* When attached to an interpolation variable, the type of the
+  **`interpolation_parameters`** attribute is a string whose value is
+  list of blank separated word pairs in the form **`term: var`**. The
+  legal values **`term`** are contained in Appendix J for each valid
+  **`interpolation_name`**. The values of **`var`** must be
+  interpolation parameter variables that exist in the file.
```

Comment:
```suggestion
  list of blank separated word pairs in the form **`term: var`**. If 
  the interpolation variable has an **``interpolation_name``**
  attribute with a valid value, then the legal values for **`term`**
  are listed in the section of Appendix J dedicated to the
  interpolation method identified by the
  **``interpolation_name``** attribute. The values of **`var`**
  must be interpolation parameter variables that exist in the file.
```

Reply:
Hi @davidhassell, I support this change from Sylvain. Would you agree?